sdk/keyvault/test-resources/test-resources.json

```diff
@@ -0,0 +1,552 @@
+      "properties": {
+        "forceUpdateTag": "[parameters('utcValue')]",
+        "azPowerShellVersion": "5.0",
+        "timeout": "PT30M",
+        "arguments": "[format(' -vaultName {0} -certificateName {1} -subjectName {2}', variables('kvName'), parameters('certificateName'), parameters('subjectName'))]",
+        "scriptContent": "param([string] [Parameter(Mandatory=$true)] $vaultName, [string] [Parameter(Mandatory=$true)] $certificateName, [string] [Parameter(Mandatory=$true)] $subjectName)\n\n$ErrorActionPreference = 'Stop'\n$DeploymentScriptOutputs = @{}\n$policy = New-AzKeyVaultCertificatePolicy -SubjectName $subjectName -IssuerName Self -ValidityInMonths 12 -Verbose\n\nAdd-AzKeyVaultCertificate -VaultName $vaultName -Name $certificateName -CertificatePolicy $policy -Verbose\n\n$newCert = Get-AzKeyVaultCertificate -VaultName $vaultName -Name $certificateName\n$tries = 0\n\ndo {\n    Write-Host 'Waiting for certificate creation completion...'\n    Start-Sleep -Seconds 10\n    $operation = Get-AzKeyVaultCertificateOperation -VaultName $vaultName -Name $certificateName\n    $tries++\n\n    if ($operation.Status -eq 'failed') {\n        throw 'Creating certificate $certificateName in vault $vaultName failed with error $($operation.ErrorMessage)'\n    }\n\n    if ($tries -gt 120) {\n        throw 'Timed out waiting for creation of certificate $certificateName in vault $vaultName'\n    }\n} while ($operation.Status -ne 'completed')\n\n$DeploymentScriptOutputs['certThumbprint'] = $newCert.Thumbprint\n$newCert | Out-String\n",
+        "cleanupPreference": "OnSuccess",
+        "retentionInterval": "P1D"
+      }
+    }
+  ],
+  "outputs": {
+    "AZURE_KEYVAULT_ENDPOINT": {
+      "type": "string",
+      "value": "[reference(variables('kvName')).vaultUri]"
+    },
+    "AZURE_MANAGEDHSM_ENDPOINT": {
+      "type": "string",
+      "condition": "[parameters('enableHsm')]",
+      "value": "[reference(variables('hsmName')).hsmUri]"
+    },
+    "AZURE_KEYVAULT_TENANT_ID": {
+      "type": "string",
+      "value": "[parameters('tenantId')]"
+    },
+    "AZURE_KEYVAULT_CLIENT_ID": {
+      "type": "string",
+      "value": "[parameters('testApplicationId')]"
+    },
+    "AZURE_KEYVAULT_CERTIFICATE_NAME": {
+      "type": "string",
+      "value": "[parameters('certificateName')]"
+    },
+    "AZURE_KEYVAULT_URI": {
+      "type": "string",
+      "value": "[reference(variables('kvName')).vaultUri]"
+    },
+    "KEYVAULT_SKU": {
+      "type": "string",
+      "value": "[reference(parameters('baseName')).sku.name]"
+    },
+    "CLIENT_OBJECTID": {
+      "type": "string",
+      "value": "[parameters('testApplicationOid')]"
+    },
+    "KEYVAULT_STORAGE_ENDPOINT_SUFFIX": {
+      "type": "string",
+      "value": "[parameters('storageEndpointSuffix')]"
+    },
+    "BLOB_STORAGE_ACCOUNT_NAME": {
+      "type": "string",
+      "value": "[variables('primaryAccountName')]"
+    },
+    "BLOB_PRIMARY_STORAGE_ACCOUNT_KEY": {
```

Comment:
Are we keeping this around because we support SAS tokens in the backup clients? Is there a plan to phase these out?